assets/src/components/demo/Demo.tsx

```diff
@@ -34,7 +34,7 @@ class Demo extends React.Component<Props, State> {
           title="Welcome to Papercups!"
           subtitle="Ask us anything in the chat window below 😊"
           primaryColor={colors.primary}
-          accountId="eb504736-0f20-4978-98ff-1a82ae60b266"
+          accountId="22af71fc-26b4-4c39-9425-e6e427be0b96"
```

Comment:
can we change this back? it's currently hardcoded to match our account in prod so that if anyone is trying out the demo page, we'll get pinged

(for testing locally i just update my account id to match this in my dev database)

Reply:
Fixed